src/app/users-signed/users-signed.component.css

```diff
@@ -73,6 +73,11 @@ mat-footer-row mat-footer-cell {
     overflow: hidden;
     vertical-align: middle;
 }
+<<<<<<< Updated upstream
```

Comment:
merge issue?

Reply:
this happened as i kept committing to the wrong branch.
I caught the first time. Sorted now